plugins/help.py

```diff
@@ -21,34 +22,42 @@ def loadJSON(jf):
 builtins = loadJSON(builtinfile)
 
 # Makes a list of all avaliable commands (from json file)
+
+
 def cmdList(cmds):
     cmdList = ""
     for cmd in cmds:
         cmdList += "**+" + cmd + ":** " + cmds[cmd]['description'] + "\n"
     return cmdList
 
 # Finds command info (from json file)
+
+
 def cmdLookup(cmds):
     lookup = ""
     for cmd in cmds:
         if (arg == cmd):
             if (plugins[cmd]['arguments'] is None):
-                cmdargs = "This command takes no arguments"
+                info = "This command takes no arguments"
             else:
-                cmdargs = ""
+                info = ""
                 for cmdarg in plugins[cmd]['arguments']:
-                    cmdargs += "`[" + cmdarg + "]`: " + plugins[cmd]['arguments'][cmdarg]['msg'] + \
-                                ". Required? " + str(plugins[cmd]['arguments'][cmdarg]['required']) + "\n"
-            lookup = "**" + cmd + "**: " + plugins[cmd]['description'] + "\n" + cmdargs
+                    info = "`[" + cmdarg + "]`: " + \
```

Comment:
i know i said to use `+` before but that was for js
in python you should use the {} operator and format as follows
`foo = "{} {}!".format("hello", "world"`